src/test/store/actions.test.js

```diff
@@ -239,44 +219,6 @@ describe('selectors/getFlameGraphTiming', function() {
   });
 });
 
-describe('selectors/getCallNodeMaxDepthForStackChart', function() {
```

Comment:
there's no test for `getCallNodeMaxDepth`, could you please add a quick one?

Reply:
#724